channeld/channeld.c

```diff
@@ -511,13 +511,14 @@ static void check_mutual_splice_locked(struct peer *peer)
 }
 
 /* Our peer told us they saw our splice confirm on chain with `splice_locked`.
- * If we see it to we jump into tansitioning to post-splice, otherwise we mark
+ * If we see it to we jump into transitioning to post-splice, otherwise we mark
  * a flag and wait until we see it on chain too. */
 static void handle_peer_splice_locked(struct peer *peer, const u8 *msg)
 {
 	struct channel_id chanid;
+	struct bitcoin_txid splice_txid;
 
-	if (!fromwire_splice_locked(msg, &chanid))
+	if (!fromwire_splice_locked(msg, &chanid, &splice_txid))
```

Comment:
we're not checking it?

Reply:
Added a check for it in [e1d7602](https://github.com/ElementsProject/lightning/pull/8021/commits/e1d760245d689163efd73b61f93e2d5d99dd4b39)